VideoApp/VideoApp/NewCode/Views/RoomToolbar/RoomToolbar.swift

```diff
@@ -36,7 +36,7 @@ struct RoomToolbar<Content>: View where Content: View {
 struct RoomToolbar_Previews: PreviewProvider {
     static var previews: some View {
         RoomToolbar {
-            RoomToolbarButton(image: Image(systemName: "arrow.left"), role: .destructive)
+            RoomToolbarButton(image: Image(systemName: "phone.down.fill"), role: .destructive)
```

Comment:
You might consider extracting reused image name strings to an enum.

Reply:
I'm thinking leave this for now and think about it more but I could be convinced otherwise also. 

There is really only 1 place in production code that uses this image. And so there is no risk to any production bug. The other 2 places are for UI previews and really any image can be used. Maybe I should have even used an image that was clearly for testing like `questionmark`.

Reply:
Thinking about this more...I may switch the previews to something like `questionmark` just to be clearer.

Reply:
I'm ok leaving it as is since it's not production code.